### PR TITLE
Ab#135117 fix attachment delete error

### PR DESCRIPTION
**Pull request by TC-Cory**

### CODE REVIEW

File: src/CSF.SRDashboard.Client/Components/DocumentInformation.razor [hunk 1]
```diff
@@ -99,18 +99,45 @@ else
...
-                        </FormInputSelect>
-                        <FormInputTextArea Id="InputTextArea" Label="@Localizer["Description"]" Rows="2" Cols="50" @bind-Value="@this.UploadedDocuments[i].Description" ValidationFor="@(() => this.UploadedDocuments[i].Description)" />
-                    </div>
-                </div>
+                        @if (this.Languages != null)
+                        {
+                            <FormInputSelect Id="InputSelect" Label="@Localizer["DocumentLanguage"]" IsRequired="true" Required="required" @bind-Value="this.UploadedDocuments[i].Language" ValidationFor="@(() => this.UploadedDocuments[i].Language)" SelectLabel=@Localizer["SelectLanguage"] ShowDefaultOption="ShowDefaultOption">
+                                @foreach (var item in Languages)
+                                {
+                        <option value="@item.Id">@Localizer[item.Text]</option>
```
Comment (TCPrimedPaul) [inline src/CSF.SRDashboard.Client/Components/DocumentInformation.razor:107]: Indentations seem to be not aligned in this code block

File: src/CSF.SRDashboard.Client/Components/DocumentInformation.razor.cs [hunk 2]
```diff
@@ -139,18 +162,46 @@ private async void InitializeDocumentTypes()
...
+        public void InitializeDelete(int index)
         {
+            SwitchModalState();
             if (index <= -1)
             {
                 return;
             }
+            this.Index = index;
+        }
+        public void SwitchModalState()
```
Comment (TCPrimedPaul) [inline src/CSF.SRDashboard.Client/Components/DocumentInformation.razor.cs:176]: I would add a space between methods for readability.

File: src/CSF.SRDashboard.Client/Components/DocumentInformation.razor [hunk 3]
```diff
@@ -75,7 +75,7 @@ else
                         <div class="mr-3">
                             @if (!this.IsReadOnly)
                             {
-                                <button class="small-buttons" @onclick="@(e => RemoveAttachment(i))">
+                                <button class="small-buttons" @onclick="@(e => InitializeDelete(i))">
```
Comment (TCPrimedPaul) [inline src/CSF.SRDashboard.Client/Components/DocumentInformation.razor:78]: InitializeDelete is kind of vague, I went to look at method and it seems to be resetting an index following a deletion. Suggestion for name: ResetIndexAfterDelete()

File: src/CSF.SRDashboard.Client/appsettings.ncd.json [hunk 4]
```diff
@@ -9,7 +9,7 @@
     "MarineMedicalGroupId": "fcc8a558-2e87-495b-9f38-718763983504"
   },
   "DocumentServiceAPI": {
-    "BaseUrl": "https://tc-api-management-dev.azure-api.net/dms/api/v1",
+    "BaseUrl": "https://ncdsafsecsurapp-document-management-service.azurewebsites.net/api/v1",
```
Comment (TCPrimedPaul) [inline src/CSF.SRDashboard.Client/appsettings.ncd.json:12]: Did the api management url stop working?

**Review by TCPrimedPaul — COMMENTED** [review]
Great work!

**Review by tc-chris — APPROVED** [review]
Tested and works good

**Pull request merged**